crates/gateway-types/src/reply.rs

```diff
@@ -773,3 +773,26 @@ mod tests {
         }
     }
 }
+
+// Types used for web socket subscription events
+pub mod ws_subscriptions {
```

Comment:
is there a logical reason to put this type in the gateway-types/src/reply.rs? From the discussions with Pathfinder, we should try to keep every code related to websocket decoupled as much as possible so for example create a websocket.rs in the gateway-types